analysis/measures_definition.py

```diff
@@ -117,71 +120,64 @@
     default="missing",
 )
 
-# Define population to be registered and above 18 years old
-denominator = has_registration & (age > 18)
-
-# Define monthly measure
-measures.define_measure(
-    name="virtual_consultations_pre_monthly",
-    numerator=has_virtual_consultation,
-    denominator=denominator,
-    group_by={"age_greater_equal_65": age_greater_equal_65},
-    intervals=months(6).starting_on("2019-04-01"),
-)
-
-measures.define_measure(
-    name="appointments_pre_monthly",
-    numerator=has_appointment,
-    denominator=denominator,
-    group_by={"age_greater_equal_65": age_greater_equal_65},
-    intervals=months(6).starting_on("2019-04-01"),
-)
-
-measures.define_measure(
-    name="virtual_consultations_during_monthly",
-    numerator=has_virtual_consultation,
-    denominator=denominator,
-    group_by={"age_greater_equal_65": age_greater_equal_65},
-    intervals=months(6).starting_on("2020-04-01"),
-)
-
-measures.define_measure(
-    name="appointments_during_monthly",
-    numerator=has_appointment,
-    denominator=denominator,
-    group_by={"age_greater_equal_65": age_greater_equal_65},
-    intervals=months(6).starting_on("2020-04-01"),
-)
+# Define population denominator
+denominator = has_appointment
```

Comment:
now you're no longer including the age restriction and registration in your population, I'm not sure this is what you want to do. you probably want:

denominator = has_registration & (age > 18) & has_appointment

Reply:
This very helpful!

Reply:
I am not sure what naming convention will be appropriate. I have changed the denominator accordingly.